collections/without_all.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+
+/**
+ * Returns an array excluding all given values.
+ *
+ * Example:
+ *
+ * ```ts
+ * import { withoutAll } from "./without_all.ts";
+ *
+ * const withoutList = withoutAll([2, 1, 2, 3], [1, 2]);
+ *
+ * ```
+ */
+export function withoutAll<T>(
+  array: Array<T>,
+  values: Array<T>,
+): Array<T> {
```

Comment:
```suggestion
export function withoutAll<T>(array: readonly T[], values: readonly T[]): T[] {
```
See https://github.com/denoland/deno_std/pull/1119

Reply:
Good point, I missed that. I am clearly too used to read `Array<T>` :-)